indico/modules/events/timetable/client/js/actions.ts

```diff
@@ -174,13 +174,33 @@ export function toggleDraft() {
   return {type: TOGGLE_DRAFT};
 }
 
-export function resizeEntry(
-  date: string,
-  id: string,
-  duration: number,
-  sessionBlockId?: string
-): ResizeEntryAction {
-  return {type: RESIZE_ENTRY, date, id, duration, sessionBlockId};
+export function resizeEntry(entry, eventId, duration, date) {
+  let entryURL: string;
+
+  switch (entry.type) {
+    case EntryType.Break:
+      entryURL = breakURL({event_id: eventId, break_id: entry.objId});
+      break;
+    case EntryType.SessionBlock:
+      entryURL = sessionBlockURL({event_id: eventId, session_block_id: entry.objId});
+      break;
+    default:
+      entryURL = contributionURL({event_id: eventId, contrib_id: entry.objId});
+  }
```

Comment:
Just an FYI, but we also use this kind of format in `EntryPopup.tsx` for example:

```js
    const editURL = {
      [EntryType.Contribution]: contributionURL({event_id: eventId, contrib_id: objId}),
      [EntryType.SessionBlock]: sessionBlockURL({event_id: eventId, session_block_id: objId}),
      [EntryType.Break]: breakURL({event_id: eventId, break_id: objId}),
    }[type];
    ```
    
    Personally I like that approach more because we start right away with a const variable

Reply:
We could also just put the URL in the original data coming from the server instead of building it locally... this would also be nice when someone gets timetable data for their own scripts etc.

FWIW The one thing I dislike a bit with the approach here is that you build all 3 URLs which just works be cause they happen to use the same input values. But it does look nicer than the switch. Moving it into a `getEntryURL()` util would also be an option of course...